### ISSUE TITLE
Color contrast fails for the text

### ISSUE THREAD

**Issue by Aparan138** [sachinmegabytes/istanbullore-staging#4]
Color contrast fails for the text. Normal criteria for the Text is 4.5:1

<img width="959" alt="Image" src="https://github.com/user-attachments/assets/d8d32b0f-5e85-4446-9c7b-7a5b37e229e2" />
